event_manager.py

```diff
@@ -60,45 +78,49 @@ def handle_event(event_type, event):
     event handler for the given event type.
     """
     event_detail = event['event']
+    event_subtype = event_detail.get('subtype')
     channel_id = event_detail['channel']
 
+    em_logger.info('Processing message with event type %s and subtype %s',
+            event_type, event_subtype)
+
     # Ensure that the message we got is not from the bot itself
-    if event_type == 'message' and event_detail.get('subtype') != 'bot_message':
+    if event_type == 'message' and event_subtype != 'bot_message':
         # Prevent users from ++ or -- themselves.
         sending_usr = event_detail.get('user')
         message = event_detail.get('text', '')
         if (sending_usr and sending_usr in message and
                 (increment_regex.match(message) or decrement_regex.match(message))):
-            print('Skipping self bump.')
+            em_logger.debug('Skipping self bump.')
             karmaBot.chastise(('++' in message), channel_id)
             return make_response('Got a self bump', 201)
 
         if message:
-            print(message)
             if increment_regex.match(message):
                 karmaBot.increment(clean_up_message(message), channel_id)
                 return make_response('Got an increment message', 201)
             if decrement_regex.match(message):
                 karmaBot.decrement(clean_up_message(message), channel_id)
                 return make_response('Got a decrement message', 201)
-            print('no regex match')
+            em_logger.debug('no regex match')
 
+    em_logger.debug('Unhandled message event type/subtype or no regex match')
```

Comment:
Can we make this more specific? Clearly we know when one or the other happens, so we should call out which one actually happened.